_sass/main.scss

```diff
@@ -0,0 +1,123 @@
+/* Global Styles */
+/* mixins, colors, etc. */
+
+$cfd-red: #E14E54;
+$navy: #181E34;
+$white: white;
+$light-grey: #C4C4C4;
+
+$breakpoints: (
+  'small':  ( min-width:  767px ),
```

Comment:
not a big deal, but small width should be 768px https://getflywheel.com/layout/css-breakpoints-responsive-design-how-to/

Reply:
You're right. I copied that scss variable from somewhere and only played with it to confirm it was being processed correctly. I will update this.